griptape/artifacts/image_artifact.py

```diff
@@ -1,23 +1,38 @@
 from __future__ import annotations
 
+import base64
+
 from attrs import define, field
 
-from griptape.artifacts import MediaArtifact
+from griptape.artifacts import BlobArtifact
 
 
 @define
-class ImageArtifact(MediaArtifact):
-    """ImageArtifact is a type of MediaArtifact representing an image.
+class ImageArtifact(BlobArtifact):
+    """Stores image data.
 
     Attributes:
-        value: Raw bytes representing media data.
-        media_type: The type of media, defaults to "image".
-        format: The format of the media, like png, jpeg, or gif.
-        name: Artifact name, generated using creation time and a random string.
-        model: Optionally specify the model used to generate the media.
-        prompt: Optionally specify the prompt used to generate the media.
+        value: The image data.
+        format: The format of the image data. Defaults to "png".
+        width: The width of the image.
+        height: The height of the image
     """
 
-    media_type: str = "image"
+    value: bytes = field(metadata={"serializable": True})
```

Comment:
does this need the bytes converter here?

Reply:
Added.

Reply:
Added